### PR TITLE
Add verifiers to CHAPI playground test suite.

### PR DESCRIPTION
**Pull request by brianorwhatever**
image

### CODE REVIEW

File: tests/assertions.js [hunk 1]
```diff
@@ -83,3 +86,48 @@ function _shouldHaveClaims({subject}) {
     'Expected credentialSubject to have at least one claim.'
   );
 }
+
+export function shouldBeValidVerification({verification}) {
+  verification.should.be.an(
+    'object',
+    'Expected the verification result to be an object.'
+  );
```
Comment (dlongley) [inline tests/assertions.js:94]:
```suggestion
  verification.should.be.an(
    'object',
    'Expected the verification result to be an object.');
```

File: tests/assertions.js [hunk 2]
```diff
@@ -83,3 +86,48 @@ function _shouldHaveClaims({subject}) {
...
+  verification.should.have.property('presentationResult');
+  verification.presentationResult.should.have.property('verified');
+  verification.should.have.property('checks');
+  verification.should.have.property('credentialResults');
+  verification.credentialResults.should.be.a(
+    'array',
+    'Expected `credentialResults` to be an array.'
+  );
```
Comment (dlongley) [inline tests/assertions.js:102]:
```suggestion
  verification.credentialResults.should.be.a(
    'array',
    'Expected `credentialResults` to be an array.');
```

File: tests/mock.data.js [hunk 3]
```diff
@@ -6,6 +6,12 @@ import {klona} from 'klona';
 import {nanoid} from 'nanoid';
 const require = createRequire(import.meta.url);
 const validVc = require('./credential.json');
+const VPs = {
+  'vp-bound': require('../fixtures/vp-bound.json'),
+  'vp-unbound': require('../fixtures/vp-unbound.json'),
```
Comment (dlongley) [inline tests/mock.data.js:11]: As mentioned elsewhere, we should avoid 'bound' language with VCs.

**Review by dlongley — APPROVED** [review]
A couple of style / naming nits here, but once resolved we should merge this as a good baseline for verifier work. Thanks!

**Comment by brianorwhatever** [comment]
Made changes to bound language.

**Comment by brianorwhatever** [comment]
@dlongley @gannan08 I have added details to the README about how to implement an issuer/verifier and the endpoints that are required.

**Review by msporny — APPROVED** [review]

**Pull request merged**